frontend/src/home/Home.tsx

```diff
@@ -564,6 +468,20 @@ class HomeComponent extends React.Component<Props, HomeState> {
   };
 
   render() {
+    if (!this.state.fetchedPlan) {
```

Comment:
To air on the safe side, just render a loading indicator if currently fetching user plans